plugins/woocommerce-blocks/assets/js/blocks/coming-soon/index.tsx

```diff
@@ -135,15 +139,12 @@ registerBlockType( metadata, {
                 body .is-layout-constrained > .woocommerce-coming-soon-banner.alignwide {
                     max-width: 820px;
                 }
-                .coming-soon-is-vertically-aligned-center:not(.block-editor-block-list__block) {
-                    position: absolute;
-                    top: 50%;
-                    transform: translateY(-50%);
-                    margin-block-start: 0;
+                .coming-soon-is-vertically-aligned-center {
                     width: 100%;
+                    align-items: stretch;
                 }
-                .woocommerce-coming-soon-header, .coming-soon-cover .wp-block-cover__background {
-                    background-color: ${ color } !important;
```

Comment:
With the Coming soon block selected, the background color is not longer respecting the selection

<img width="289" alt="image" src="https://github.com/woocommerce/woocommerce/assets/1922453/8addb2a8-07cf-4178-805c-1c31fb19728a">

Reply:
Good catch!

Reply:
Fixed in https://github.com/woocommerce/woocommerce/pull/46914/commits/c471e520d322b7073fa4ae854d7113be0b7c92a5